src/v3/src/transformer/main.ts

```diff
@@ -37,6 +38,7 @@ const logger: TransformStepFn = (formbag) => {
 
 export const transformIdxTransaction = (options: TransformationOptions): FormBag => {
   const transformationStepFns: TransformStepFn[] = [
+    transformTransactionData(options),
```

Comment:
This transformer allows us to update any transaction related data before it flows to the subsequent layout transformers. This is key for scenarios like IDP/PIV because like V2, some client-side transformation has to occur first (i.e. updating remediation names)